src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/_payments.html.twig

```diff
@@ -1,13 +1,21 @@
 {% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
 
+{% set state = order.paymentState %}
+{% set value = 'sylius.ui.' ~ state %}
+
+{% include "@SyliusShop/Common/Order/Label/PaymentState/orderPaymentState.html.twig" %}
```

Comment:
We should check the state to not display it for **cart** `{% if state != 'cart' %}`

Reply:
I'd say that the `if` should be there, not in the included template (we just don't want to have **anything** there if the state is `cart` :)). It refers also to `singlePaymentState` inclusion